test/e2e/common/container_probe.go

```diff
@@ -50,9 +50,9 @@ var _ = framework.KubeDescribe("Probing container", func() {
 	})
 
 	/*
-		    Testname: pods-readiness-probe-initial-delay
-		    Description: Make sure that pod with readiness probe should not be
-			ready before initial delay and never restart.
+		Release : v1.9
+		Testname: Pod readiness probe with initial delay
+		Description: Create a Pod that is configured with a initial delay set on the readiness probe. Check the Pod Start time to compare to the initial delay. The Pod MUST be ready only after the specified initial delay.
```

Comment:
Nit: need a consistent voice in these test descriptions, ie:
- "Create a Pod that is configured with"
- "Pod is created with"
- " A Pod is created with"

Reply:
Fixed